utils/index.js

```diff
@@ -21,8 +21,14 @@ const titleCase = str => {
   }
 };
 
+const firstSentenceOnly = p => {
+  const idx = p.search(/(\?|\.|!)/);
+  return p.slice(0, idx + 1);
+};
+
```

Comment:
I like your line of thinking here, but I'm not sure if you can rely on the fact that a sentence will exist proper in whatever a user chooses to put in a particular field. @CB6670 mentioned the idea of tourniquet the length of a card is just to take a slice of the string up to a defined point and append '...' to the end. I kind of agree with him in this case.
return str.slice(0,25) + '...';

Reply:
What about attempting to find the index of a sentence ending punctuation mark, then if it does not exist, using a predetermined length?

Reply:
I think that would totally work also man.

Reply:
Or is the worry that a punctuation mark might occur too far into the text?

Reply:
Maybe to far in the text, maybe there is no punctuation at all but a really long body of text.